fretboard-scheduler-jobscheduler/src/main/java/mozilla/components/service/fretboard/scheduler/jobscheduler/JobSchedulerSyncScheduler.kt

```diff
@@ -0,0 +1,45 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.service.fretboard.scheduler.jobscheduler
+
+import android.app.job.JobInfo
+import android.app.job.JobScheduler
+import android.content.ComponentName
+import android.content.Context
+import java.util.concurrent.TimeUnit
+
+/**
+ * Class used to schedule sync of experiment
+ * configuration from the server
+ */
+class JobSchedulerSyncScheduler(context: Context) {
+    private val jobScheduler = context.getSystemService(Context.JOB_SCHEDULER_SERVICE) as JobScheduler
+
+    /**
+     * Schedule sync with the constrains specified
+     *
+     * @param jobInfo object with the job constraints
+     */
+    fun schedule(jobInfo: JobInfo) {
+        jobScheduler.schedule(jobInfo)
+    }
+
+    /**
+     * Schedule sync with the default constraints
+     * (once a day)
+     *
+     * @param serviceName object with the service to run
+     */
+    fun schedule(serviceName: ComponentName) {
+        val jobInfo = JobInfo.Builder(DEFAULT_JOB_ID, serviceName)
+            .setPeriodic(TimeUnit.DAYS.toMillis(1))
+            .build()
+        jobScheduler.schedule(jobInfo)
+    }
+
+    companion object {
+        private const val DEFAULT_JOB_ID = 40
```

Comment:
Can we let the consumer pass this in with schedule()? Just to avoid re-using an ID that the app already uses. We could make it an optional parameter and use 40 as default - but that may cause more problems than help. :)